packages/twenty-front/src/modules/command-menu/pages/page-layout/utils/buildChartGroupByFieldConfigUpdate.ts

```diff
@@ -51,6 +52,14 @@ export const buildChartGroupByFieldConfigUpdate = <
     'PieChartConfiguration',
   );
 
+  const fieldMetadata = objectMetadataItem?.fields?.find(
```

Comment:
Let's try to keep the naming fieldMetadataItem in the front end to stay coherent